class Failure {
  final String message;
  const Failure([this.message = 'error.unexpected']);
}
